libs/components/datetime/src/lib/modules/date-range-picker/date-range-picker.component.ts

```diff
@@ -280,7 +280,9 @@ export class SkyDateRangePickerComponent
   public helpKey: string | undefined;
 
   @HostBinding('style.display')
-  protected display: string | undefined;
+  protected get display(): 'none' | undefined {
+    return this.label || this.labelText ? undefined : 'none';
+  }
```

Comment:
i know this isn't logic that you added, but any idea why this isn't also checking if label text has been set required by the labelTextRequiredSvc?